docs/understanding-radicle/glossary.md

```diff
@@ -4,14 +4,16 @@ title: Glossary
 ---
 
 ## branch
-A parallel version of a [repository][re] that consists of changesets. A branch
-is separate from the default branch, allowing one to work freely without
-disrupting the "live" version of a [repository][re]. Once changes are made, a
-branch can be merged back into the default branch to publish changes.
+A parallel version of a [project][]. A branch is separate from the default
+branch, allowing one to work freely without disrupting the "live" version of a
+[repository][re]. Once changes are made, a branch can be merged back into the
+default branch to publish changes.
```

Comment:
I'd kill this section, `branch` is a git concept and better explained in the git docs.